### PR TITLE
fix(curriculum): Add default white bg to tree DS image for better view

### PR DESCRIPTION
**Pull request by cdadityang**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

In the [Data Structures: Add a New Element to a Binary Search Tree](https://www.freecodecamp.org/learn/coding-interview-prep/data-structures/add-a-new-element-to-a-binary-search-tree) lesson, there is this image:

image

But when the user's NIGHT MODE is ON, this image/png makes it hard to see/read the image and it looks like this.
image

So I've just added a `background` property for the default transparent image with `#f5f6f7`, and it looks way better:
image

Also, this doesn't have any affect on users learning with LIGHT THEME as the bg is same.

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/add-a-new-element-to-a-binary-search-tree.english.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ forumTopicId: 301618
...
-<div style='width: 100%; display: flex; justify-content: center; align-items: center;'><img style='width: 100%; max-width: 350px;' src='https://user-images.githubusercontent.com/18563015/32136009-1e665d98-bbd6-11e7-9133-63184f9f9182.png'></div>
+<div style='width: 100%; display: flex; justify-content: center; align-items: center;'><img style='width: 100%; max-width: 350px; background: #f5f6f7;' src='https://user-images.githubusercontent.com/18563015/32136009-1e665d98-bbd6-11e7-9133-63184f9f9182.png'></div>
```
Comment (Sky020) [inline curriculum/challenges/english/10-coding-interview-prep/data-structures/add-a-new-element-to-a-binary-search-tree.english.md:14]:
```suggestion
<div style='width: 100%; display: flex; justify-content: center; align-items: center;'><img style='width: 100%; max-width: 350px; background-color: var(--gray-05);' src='https://user-images.githubusercontent.com/18563015/32136009-1e665d98-bbd6-11e7-9133-63184f9f9182.png'></div>
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Something more verbose.

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/add-a-new-element-to-a-binary-search-tree.english.md (hunk 1 above)
Comment (cdadityang) [reply curriculum/challenges/english/10-coding-interview-prep/data-structures/add-a-new-element-to-a-binary-search-tree.english.md:14]: `var` CSS variables are not supported in old browsers. So I guess `background-color: #f5f6f7;` would be fine?

Comment (cdadityang) [reply curriculum/challenges/english/10-coding-interview-prep/data-structures/add-a-new-element-to-a-binary-search-tree.english.md:14]: I see the codebase uses CSS vars, I'll make that change.

Comment (ojeytonwilliams) [reply curriculum/challenges/english/10-coding-interview-prep/data-structures/add-a-new-element-to-a-binary-search-tree.english.md]: We already use `var` in various places, so don't worry about using it here.

**Comment by cdadityang** [comment]
@Sky020 any updates?

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 . Just need a second approving review, before this is merged.

**Review by Manish-Giri — APPROVED** [review]
LGTM

**Review by thecodingaviator — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
@cdadityang Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :memo:

**Comment by cdadityang** [comment]
@ojeytonwilliams Cool. Thanks

**Pull request merged**